### PR TITLE
small microoptimization for `SIGNAL_ADDTRAIT` and `SIGNAL_REMOVETRAIT`

### PR DESCRIPTION
**Pull request by Absolucy**
## About The Pull Request

iirc, string + string is const folded at compile-time, while "whatever [string]" is not.

## Why It's Good For The Game

extra performance for practically no cost? why not

## Changelog
:cl:
code: Very slightly improved the performance of code related to adding and removing traits.
/:cl:

### CODE REVIEW

File: code/__DEFINES/traits/_traits.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
-#define SIGNAL_ADDTRAIT(trait_ref) "addtrait [trait_ref]"
-#define SIGNAL_REMOVETRAIT(trait_ref) "removetrait [trait_ref]"
+#define SIGNAL_ADDTRAIT(trait_ref)		("addtrait " + trait_ref)
+#define SIGNAL_REMOVETRAIT(trait_ref)	("removetrait " + trait_ref)
```
Comment (LemonInTheDark) [inline code/__DEFINES/traits/_traits.dm:2]: what's goin on with the indentation?

**Comment by LemonInTheDark** [comment]
profileable? or nah. am ok taking this on faith given our understanding of + vs []

File: code/__DEFINES/traits/_traits.dm (hunk 1 above)
Comment (Absolucy) [reply code/__DEFINES/traits/_traits.dm:2]: i just made them line up

**Comment by Absolucy** [comment]
probably not, but it doesn't hurt. was testing this downstream and it worked just fine, after fixing a `ADD_TRAIT(thing, list(...), source)`

File: code/__DEFINES/traits/_traits.dm (hunk 1 above)
Comment (MrMelbert) [reply code/__DEFINES/traits/_traits.dm:2]: Doing this is banned in our style guide

Comment (Absolucy) [reply code/__DEFINES/traits/_traits.dm:2]: oh, okey. apologies

**Comment by LemonInTheDark** [comment]
❤️

**Pull request merged**